tools/test-case-generators/generate-all-test-cases

```diff
@@ -1290,9 +1290,9 @@ class ManifestTestCaseMatrixGenerator(BaseTestCaseMatrixGenerator):
                     script_dir = os.path.dirname(__file__)
                     log.info("Generating manifest for %s-%s-%s", distro_name, arch_name, image_type)
                     try:
-                        subprocess.check_call([f"{script_dir}/generate-test-cases", "--distro", f"{distro_name}", "--arch",
+                        subprocess.run([f"{script_dir}/generate-test-cases", "--distro", f"{distro_name}", "--arch",
                             f"{arch_name}", "--image-types", f"{image_type}", "--store", "/dev/null", "--output", f"{self.output}",
-                            "--keep-image-info"], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
+                            "--keep-image-info"], stdout=subprocess.PIPE, stderr=subprocess.PIPE, check=True, encoding="utf-8")
```

Comment:
The commit message for this change seems to have been copied from 8ee90334bab71920884914963aef6e20c3e26a9d and doesn't match the change.

Reply:
I cherry-picked that change and submitted is as a separate PR, which has been already merged. I think that the change makes sense, because if the generation fails, there won't be any output. However, the commit message should be definitely adjusted.

Reply:
I think at some point this was part of that commit. @thozza, can you tell me a better commit message pls?

Reply:
I'll change it. But I have no idea of the context of this change. What would be an appropriate message?

Reply:
something like "Capture stdout and stderr output when running `generate-test-cases` in the `manifests` command. This is helpful for debugging test case generation failures."

Reply:
Thanks! done